README.md

```diff
@@ -1 +1,18 @@
 # ood-pyraf-apptainer
+This repo is based off of [ood-remote-desktop-rosgazebo](https://github.com/Harvard-ATG/ood-remote-desktop-rosgazebo) and is used to create a remote desktop environment with pyraf, iraf and DS9 packages installed via conda.
+
+## Installation
+In order to install the app
+
+Note you will need to be in the head node while building any of the apptainer images.
+Apptainer images should be located `/shared/apptainerImages`
+
+- Activate the spack environment and then use activate the apptainer env.
+```bash
+root@ip:/# . shared/spack/share/spack/setup-env.sh
+root@ip:/# spacktivate apptainer
+```
+- Build the image from the `pyraf.def` file by running `apptainer build pyraf.sif pyraf.def` Note: the `pyraf.sif` should be available in the `/shared/apptainerImages` folder
+
+- After the image is successful you can add the app by following the [Adding OOD app runbook](https://docs.google.com/document/d/1LejSmmgY7rJDBwl1gl-Q835lSbf6v2b2J3TFhxPNwfc/edit?tab=t.0#heading=h.n0uvlxdrj7e3)
```

Comment:
Just a note that this github repo is public, but the linked google doc is not. I don't think this is a problem, but we should be aware when publishing public repos.

Reply:
If we do publish public repos with links to non-public documents (which I also think is fine), is it worth calling out with something like `(internal HUIT documentation)` after it? I think we could also just wait for it to come up, possibly with a random access request or GH issue.

Reply:
I like the idea of adding a disclaimer for now and if folks are busting down our door for the runbook we can add it to the readme.